app/domain/SlpFileWriter.js

```diff
@@ -37,6 +46,59 @@ export default class SlpFileWriter {
 
   updateSettings(settings) {
     this.folderPath = settings.targetFolder;
+    this.obsIP = settings.obsIP;
+    this.obsSourceName = settings.obsSourceName;
+  }
+
+  async connectOBS() {
+    if (this.obsIP && this.obsSourceName) {
+      await this.obs.connect({address: this.obsIP});
+      this.scenes = await this.obs.send("GetSceneList");
```

Comment:
`const obsScenes =` would probably be better here

Reply:
But I use that in a different function.